validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -137,6 +141,7 @@ private static void addValidatorCountMetric(
 
   @Override
   protected SafeFuture<?> doStart() {
+    forkProvider.doStart().reportExceptions();
```

Comment:
This should be calling `start()`.
```suggestion
    forkProvider.start().reportExceptions();
```